libs/protocol-ui/src/stories/atoms/FormContainer.stories.tsx

```diff
@@ -0,0 +1 @@
+// TODO: need to create this component before creating a story
```

Comment:
Let's delete this Story until we have the actual component

Reply:
I just deleted this. Resolved.